### ISSUE TITLE
Will there be support for Ubuntu 19.04 any time soon?

### ISSUE THREAD

**Issue by yousuffarhan** [mail-in-a-box/mailinabox#1574]
Just wondering if Ubuntu 19.04 will be supported any time soon?

**Comment by JoshData** [comment]
No, we're at most tracking the Long Term Support (LTS) releases (https://www.ubuntu.com/about/release-cycle).

**Comment by EliterScripts** [comment]
I don't mean to bust your chops, JoshData, but we were just recently on Ubuntu 14.04, and totally ignored Ubuntu 16.04 because 14.04 was still supported by Canonical, and JoshData did not want to have two different projects to maintain simultaneously.

Since LTS is only supported for 5 years, you likely won't see Mail-in-a-Box doing a dist-upgrade until Ubuntu LTS 22.04. Since it won't be until April 2023 until Ubuntu 18.04 is dead (end of life), you may only see a mail-in-a-box Ubuntu LTS 22.04 a year after Ubuntu 22.04 is released.

That is, if mail-in-a-box continues to be developed and maintained the way it is. The other factors that go into play is that Canonical apparently is talking about extending support for Ubuntu 16.04 and 18.04 because of Internet of Things (IoT), so we may see a support life longer than 5 years from them.

You always have the option of tinkering Mail-in-a-Box to run on Ubuntu 19.04, but that would be an unsupported modification, and also may be more of a pain than it is worth, as you might have to modify more things than you think. Plus, before you know it, you'll have to switch to something else because 19.04 will die quickly, and have to tinker Mail-in-a-Box a second time.

**Comment by filips123** [comment]
What is actually needed to change to support a specific version? Won't version for Ubuntu 18.04 also work on 19.04?

**Comment by JoshData** [comment]
There's no way to know until you research what's changed in Ubuntu and how that will impact Mail-in-a-Box, and then implement it and test it out for a few weeks, since it's impossible to know how things really work until you do it. Then on top of it, if there are differences, we have to provide support for those differences in the code, documentation, and in answering questions on the forum, which is more time for the maintainers.

**Comment by alento-group** [comment]
@yousuffarhan and @filips123 

I am curious ... what advantage would there be to run MiaB on an annual release rather than a LTS release of Ubuntu?  I think that I would want to avoid the upgrading headache at all costs when humanly possible.  Can you shed some light on this for me please?

**Comment by filips123** [comment]
@alento-group Probably none. But maybe to use MiaB with other programs that need newer versions?

**Comment by alento-group** [comment]
@filips123 MiaB is designed to be utilized on a VPS dedicated to **ONLY** MiaB so there should not be any other programs installed on the server that need newer versions.  This is the official view of the developers afaik.
